chapters/data/main.tex

```diff
@@ -604,292 +597,275 @@ \subsection{Pairwise correlation}\label{subsec:diab_correlation}
-    corresponding predicted values, \(y_1, \ldots, y_n\). Then the
-    \emph{standard error of the estimate}, denoted by \(SE\), is defined to be:
-
-    \[
-        SE = \sqrt{%
-            \frac{\sum_{i=1}^{n} {\left(x_i - y_i\right)}^2}{n}
-        }
-    \]
-
-    The standard error represents the average distance (error) of the data
-    points from the regression line. The benefit of this statistic is that it
-    gives a measure of the precision of the model on the scale of the variable
-    that has been predicted.
-\end{definition}
-
-Figures~\ref{fig:admissions}~\&~\ref{fig:netcost_proportions} suggest that the
-amount of resources consumed by the diabetic population is increasing, though
-slowly. The former indicates that on average the number of diabetic patients
-visiting the hospital is increasing slowly (approximately a one percent increase
-over five years), and from the latter it is seen that the yearly average
-proportion of net spending on diabetic patients has also experienced a shallow
-increase of roughly half a percent over the same period. So, indeed, these plots
-give evidence to support the claim.
-
-In addition to this, both figures show a distinct divergence as time progresses
-as shown by both the spread in the monthly averages and the widening of the
-yearly error bars. This is an interesting phenomenon; there seems no apparent
-reason for this variability to increase in recent years with improved policy on
-prevention, diagnosis, management and treatment~\cite{NICE}.
-
-With the final figure in this section it is clear that \-- despite the slight
+Figures~\ref{fig:admissions}~---~\ref{fig:los_time} show how these quantities
+evolve over the entire data period. In each case, the weekly, monthly and yearly
+means are shown. The data has been aggregated in this way, rather than using the
+daily data, in an attempt to smooth out the misrepresentation that is described
+above. In addition to these plotted points, the data has been fitted with a
+standard least-squares linear regression model.
+
+Figures~\ref{fig:admissions}~and~\ref{fig:netcost_proportions} suggest that the
+amount of resources consumed by the diabetic population is increasing slowly.
+The former indicates that, on average, the number of diabetic patients
+visiting the hospital is increasing at a rate of approximately one percent over
+five years. From the latter, it is seen that the yearly average proportion of
+net spending on diabetic patients has also experienced a shallow increase of
+roughly half a percent over the same period.
+
+In addition to this, both figures show some form of divergence over time, as
+shown by the spread in the weekly and monthly averages. This is an interesting
+phenomenon; there seems no apparent reason for this variability to increase in
+recent years with improved policy on prevention, diagnosis, management and
+treatment~\cite{NHS:ltp,NICE,Penn2018,PHE}. The droops toward the end of the
```

Comment:
```suggestion
treatment~\cite{NHS:ltp,NICE,Penn2018,PHE}. The drops toward the end of the
```

Can you double check how your spell checker missed this @daffidwilde ?

Reply:
> Can you double check how your spell checker missed this @daffidwilde ?

lol - just realised that droops is a word...

Reply:
Are you okay with droop?

Reply:
That's a good point actually, I think drop is better :+1:

Reply:
Fixed by 89c7a11